icubam/www/handlers/upload_csv.py

```diff
@@ -1,10 +1,26 @@
 import os
+import re
 import tornado.web
 import logging
 import time
 from icubam.www.handlers import base
 
 
+def clean_path(f_name):
+  time_str = time.strftime("%Y-%m-%d-%H:%M:%S")
+  # Sanitize the name some:
+  f_name = f_name.lower()
+  f_name = os.path.normpath(f_name)
+  f_name = os.path.split(f_name)[1]
```

Comment:
are we sure there is a 2nd element ?  should we prevent from raising error here?

Reply:
path.split always returns two elements even if it has an empty [0].